pkg/node/sendoperation/callsc/callsc.go

```diff
@@ -33,18 +33,23 @@ type CallSC struct {
 
 func New(address []byte, function string, parameters []byte, gazLimit uint64, coins uint64,
 ) *CallSC {
+	// New testnet20 addresses needs a byte 0 for AU addresses and byte 1 for AS addresses
+	addressVersioned := append([]byte{byte(1)}, address...)
```

Comment:
Should be versionedAddress to stay consistent with the rest (b58VersionedCheckEncode for example)